src/benchmark_api_internal.h

```diff
@@ -14,20 +14,21 @@ namespace internal {
 
 // Information kept per benchmark we may want to run
 struct Benchmark::Instance {
-  std::string      name;
-  Benchmark*       benchmark;
-  ReportMode       report_mode;
-  std::vector<int> arg;
-  TimeUnit         time_unit;
-  int              range_multiplier;
-  bool             use_real_time;
-  bool             use_manual_time;
-  BigO             complexity;
-  BigOFunc*        complexity_lambda;
-  bool             last_benchmark_instance;
-  int              repetitions;
-  double           min_time;
-  int              threads;    // Number of concurrent threads to us
+  std::string       name;
```

Comment:
Yes and no. I added the counters (line 27), it sticked out like a sore thumb, and to make it aligned I added one space to the second column.